org.pwall.Pwall.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "org.pwall.Pwall",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
```

Comment:
GOME 40 has bee EOL for over 18month.

44 is the current one.

Reply:
I'll change the version.